### ISSUE TITLE
Input properties with a custom name are never mapped 

### ISSUE THREAD

**Issue by jsp-up** [graphql-dotnet/graphql-dotnet#1358]
## Summary
The properties of my `InputObjectGraphType` are not mapped when I give them a name.
In the example bellow `MyProp` will always be null because of the name `myCustomName`.
Did I missed something ?

## Relevant information
**ApiQuery**
```C#
 this.Field<ListGraphType<MyType>>(
        "myEndpoint",
        arguments: new QueryArguments(
        new QueryArgument<MyClassType> { Name = "args" }
        ),
        resolve: context => {
               MyClass = context.GetArgument<MyClass>("args")
        });
```

**MyClass**
```C#
    public class MyClass: IMyClass
    {
        public decimal? MyProp { get; set; }
    }
```

**MyClassType**
```C#
    public class MyClassType: InputObjectGraphType<MyClass>
    {
        public MyClassType()
        {
            this.Name = "ClassName";

            this.Field(x => x.MyProp, true, type: typeof(IdGraphType)).Name("myCustomName");
        }
    }
```
**Query**
```C#
query {
  myEndpoint(args: {myCustomName: 1}) {
}
```

### Environment (if relevant)
DotNetCore 2.2, GraphQL .NET 2.4, Windows 7.

**Comment by sungam3r** [comment]
This is a known behavior for input types. You are not the first to encounter this. I already had thoughts on what could be improved. Fixing this issue is in my personal todo list.

**Comment by sungam3r** [comment]
Relates to #1045

**Comment by sungam3r** [comment]
Relates to #1176

**Comment by jsp-up** [comment]
OK thanks, I'll wait for this feature !

**Comment by sungam3r** [comment]
@jsp-up looking through old tasks i found that i already fixed this in #1107. Will be merged soon.

**Comment by sungam3r** [comment]
> OK thanks, I'll wait for this feature !

@jsp-up merged!
